tests/xfstests/install.pm

```diff
@@ -45,6 +45,16 @@ sub run {
     zypper_call('--gpg-auto-import-keys ref', timeout => 600);
     zypper_call('in qa_test_xfstests',        timeout => 1200);
 
+    # Install upstream fio if not find usable fio in current repos
+    if (script_run('zypper se fio')) {
```

Comment:
script_run() returns the return value of the command. http://open.qa/api/testapi/#_script_run

In this case, if 'fio' package exists, zypper will return "0" and it won't hit the condition..

I would suggest something like:

    if (script_run('zypper se fio') == 0)

and of course a test run to verify it installs from repo.

Reply:
You are quite right, I corrected it.

Reply:
I found another issue, if use `zypper se fio` will also list qa_test_fio, add a -x will be perfect.
   -x, --match-exact Searches for an exact name of the package.